vue/components/ui/atoms/dropdown/dropdown.vue

```diff
@@ -105,7 +105,7 @@
     box-sizing: border-box;
     color: $dark-grey;
     font-size: 13px;
-    font-weight: 600;
+    font-weight: 400;

```

Comment:
This causes changes in all uses of dropdown. I know it matches the design of the group orders, but are you sure it also matches the design of other components that also use this component?

For example, in here - https://www.figma.com/file/QT6iYWErNNalTgUw4uujBB/RIPE-Pulse---Private?node-id=608%3A20160 - the dropdown used in pulse orders, its set as 500.
In the country group managers util in util-vue - https://www.figma.com/file/ehZVAgvkBuvaOhutQFBI0V/RIPE-Utils-Vue---Public?node-id=1892%3A2905 - its using 600